src/views/HousingLottery/HousingLottery.module.scss

```diff
@@ -1,25 +1,52 @@
-.submit_button {
-  position: "absolute";
-  margin: "auto";
-  top: 15px;
-  bottom: 0;
-  left: 45em;
-  right: 20em;
-  height: 3em;
-  width: 9em;
+.submit_preference_button {
+  height: 45px;
+  margin-left: 10px;
```

Comment:
Thank you for the review and I think what you pointed out was definitely a mistake that we have been consistently making when styling with css, however I will be deleting that specific class because it was one of the buttons that we made at the beginning of the project to test some of our components hence we are no longer in need of it. But I will be fixing all the other css files referencing the review you have given us here.

Reply:
[https://github.com/gordon-cs/gordon-360-ui/pull/2197](url)
I tried to fix all the issues that you have mentioned in the feedback. Will be merging it to housing-lottery branch soon!!